src/libraries/tests.proj

```diff
@@ -169,6 +173,10 @@
                       Exclude="@(ProjectExclusions)"
                       BuildInParallel="false"
                       Condition="'$(TestTrimming)' == 'true'" />
+    <ProjectReference Include="..\mono\netcore\sample\Android\AndroidSampleApp.csproj"
```

Comment:
same feedback about how to do this as I left in the iOS PR :)

Reply:
Removed, thanks!